pkg/sypgp/sypgp.go

```diff
@@ -349,6 +350,68 @@ func StorePubKey(e *openpgp.Entity) (err error) {
 	return
 }
 
+// compareLocalPubKey : compares a key ID with a string. return true if the
+// key / oldToken *dont* match.
+func compareLocalPubKey(e *openpgp.Entity, oldToken string) bool {
+	if fmt.Sprintf("%X", e.PrimaryKey.Fingerprint) != oldToken {
+		return true
+	}
+	return false
+}
+
+// RemovePupKey : will delete all keys with the ID of (toDelete string)
+func RemovePupKey(toDelete string) error {
```

Comment:
So... it's intentional :-)

What's a "pup key"?

Reply:
Sort for `public key`, as oppose `private key`... I think :thinking: